packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableDepositRow.tsx

```diff
@@ -190,7 +190,7 @@ export function TransactionsTableDepositRow({
 
     if (tx.depositStatus === DepositStatus.CREATION_FAILED) {
       // In case of a retryable ticket creation failure, mark only the token deposits as errors
-      return tx.asset !== 'eth'
+      return tx.asset.toLowerCase() !== 'eth'
```

Comment:
Changes like this in this PR ensure we lowercase only before comparing for values, but don't do any string manipulation while displaying token-symbols.